server/routing/lobby.go

```diff
@@ -71,6 +72,45 @@ func (lr *LobbyRoutes) lobbyJoinHandler(w http.ResponseWriter, r *http.Request)
 	return
 }
 
+func (lr *LobbyRoutes) lobbyReadyHandler(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	lobbyID := vars["lobbyID"]
+
+	playerName, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		panic(err)
+	}
+
+	lobby, err := lr.Services.Lobby.ReadyPlayer(lobbyID, string(playerName))
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusNotFound)
+		return
+	}
+
+	msg := pogo.ReadyMsg{
+		ID:      lobby.ID.String(),
+		Players: lobby.Players,
+		ReadyStatus: string(playerName) +" is "+ lobby.PlayerReady[string(playerName)],
```

Comment:
This ties in with my earlier comment. I sort of feel we should just include player readiness in our `LobbyMsg` type and use that for all lobby updates.